tf_encrypted/protocol/securenn/odd_tensor.py

```diff
@@ -160,6 +160,12 @@ def im2col(self, h_filter, w_filter, padding, strides) -> 'OddImplicitTensor':
     def conv2d(self, other, strides, padding='SAME') -> 'OddImplicitTensor':
         raise NotImplementedError()
 
+    def batch_to_space_nd(self, block_shape, crops):
```

Comment:
could we leave these out? is something complaining if we don't have them? trying to avoid this tensor type getting bloated

Reply:
I see that the odd tensor as being hidden and not for use elsewhere, so think it's okay to get hard errors :)